CHANGELOG.md

```diff
@@ -1,7 +1,30 @@
+ - Fixed numerous things in the UI. ([#3910](https://github.com/brave/browser-laptop/issues/3910))([#3916](https://github.com/brave/browser-laptop/issues/3916))([#3908](https://github.com/brave/browser-laptop/issues/3908))([#3856](https://github.com/brave/browser-laptop/issues/3856))([#3622](https://github.com/brave/browser-laptop/issues/3622))([#3673](https://github.com/brave/browser-laptop/issues/3673))([#3708](https://github.com/brave/browser-laptop/issues/3708))([#3739](https://github.com/brave/browser-laptop/issues/3739))([#3855](https://github.com/brave/browser-laptop/issues/3855))([#3629](https://github.com/brave/browser-laptop/issues/3629))([#3813](https://github.com/brave/browser-laptop/issues/3813))([#3612](https://github.com/brave/browser-laptop/issues/3621))([#3833](https://github.com/brave/browser-laptop/issues/3833))([#3677](https://github.com/brave/browser-laptop/issues/3677))([#3803](https://github.com/brave/browser-laptop/issues/3803))([#3484](https://github.com/brave/browser-laptop/issues/3484))([#3940](https://github.com/brave/browser-laptop/issues/3940))
+ - Fixed typing in the address bar. ([#3928](https://github.com/brave/browser-laptop/issues/3928))([#3927](https://github.com/brave/browser-laptop/issues/3927))([#3857](https://github.com/brave/browser-laptop/issues/3857))([#3718](https://github.com/brave/browser-laptop/issues/3718))([#3712](https://github.com/brave/browser-laptop/issues/3712))([#3933](https://github.com/brave/browser-laptop/issues/3933))
+ - Fixed Autocomplete for Twitter login. ([#3915](https://github.com/brave/browser-laptop/issues/3915))
+ - Fixed 1Password not filling the form and Brave saving the password. ([#3897](https://github.com/brave/browser-laptop/issues/3897))
+ - Fixed enabling "Web notifications" on Twitter. ([#3579](https://github.com/brave/browser-laptop/issues/3579))
+ - Fixed Brave crash when Dashlane password manager is selected. ([#3909](https://github.com/brave/browser-laptop/issues/3909))
+ - Fixed crash when Linux users would check for updates. ([#3907](https://github.com/brave/browser-laptop/issues/3907))
+ - Fixed Redhat desktop launcher. ([#3905](https://github.com/brave/browser-laptop/issues/3905))
+ - Fixed setting Brave as the default browser under Linux. ([#3776](https://github.com/brave/browser-laptop/issues/3776))
+ - Fixed privacy retention for new tabs opened via context menu actions. ([#3662](https://github.com/brave/browser-laptop/issues/3662))([#3895](https://github.com/brave/browser-laptop/issues/3895))
+ - Fixed a LastPass create account issue. ([#2936](https://github.com/brave/browser-laptop/issues/2936))
+ - Fixed renderer crash after signing into https://<subdomain>.slack.com/home. ([#3841](https://github.com/brave/browser-laptop/issues/3841))
```

Comment:
This won't be rendered properly in markdown because it'll think it's an html tag.